docs/src/main/asciidoc/container-image.adoc

```diff
@@ -118,7 +120,7 @@ On top the the global container image options, the following jib specific option
 
 On top the the global container image options, the following docker specific options are availble:
 
-.Dokcer Options
+.Docker Options
```

Comment:
We start the documentation with: no need for docker. And then we have docker options. Should we rename them?

Reply:
I went ahead and removed these lines. WDYT?

Reply:
Agreed!